bdftopcf not building due to missing xproto dep

when building on CentOS cluster :
spack install bdftopcf

fail

  >> 80    checking for BDFTOPCF... configure: error: Package requirements (xfont) were not met:
     81    
     82    Package 'xproto', required by 'xfont', not found

--- a/var/spack/repos/builtin/packages/bdftopcf/package.py
+++ b/var/spack/repos/builtin/packages/bdftopcf/package.py
@@ -41,4 +41,6 @@ class Bdftopcf(AutotoolsPackage):
     depends_on('libxfont')
 
     depends_on('pkg-config@0.9.0:', type='build')
+    depends_on('xproto', type='build')
+    depends_on('fontsproto', type='build')
     depends_on('util-macros', type='build')

